.travis.yml

```diff
@@ -7,7 +7,26 @@ python:
 before_install:
   - python3 -m pip install --upgrade pip setuptools wheel
 install:
-  - pip3 install git+https://github.com/nextstrain/cli
+  # https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/use-conda-with-travis-ci.html#the-travis-yml-file
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+      wget https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh -O miniconda.sh;
+    else
+      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+    fi
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - source "$HOME/miniconda/etc/profile.d/conda.sh"
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+  # Useful for debugging any issues with conda
+  - conda info -a
+  # Install nextstrain cli
+  - conda install -n base -c conda-forge mamba --yes
+  - conda activate base
+  - mamba create -n nextstrain -c bioconda nextstrain-cli --yes
```

Comment:
This would install the latest `nextstrain-cli` from bioconda rather than latest from the default branch (current behavior with `pip3 install git+https://github.com/nextstrain/cli`). I think this should be fine though.